system/core/base.php

```diff
@@ -664,7 +665,10 @@ function hypha_indexPages($language) {
 		// get list of available pages and sort alphabetically
 		foreach(hypha_getPageList() as $page) {
 			$lang = hypha_pageGetLanguage($page, $language);
-			if ($lang) if (isUser() || ($page->getAttribute('private')!='on')) $pageList[] = $lang->getAttribute('name').($page->getAttribute('private')=='on' ? '&#;' : '');
+			if ($lang) if (isUser() || ($page->getAttribute('private')!='on')) {
+				$pageList[] = $lang->getAttribute('name').($page->getAttribute('private')=='on' ? '&#;' : '');
+				$pageListDatatype[$lang->getAttribute('name')] = $page->getAttribute('type');
```

Comment:
This $pageListDatatype variable is introduced, but not used anywhere. It also seems unrelated to private handling, making me suspect this line belongs in a later commit?

Reply:
Also, adding this line is the only thing changed by this entire patch hunk I believe (other than whitespace changes), so the entire thing could then be moved.

Reply:
Moved $pageListDatatype to other commit